types/utils.go

```diff
@@ -80,8 +80,8 @@ func ParseTimeBytes(bz []byte) (time.Time, error) {
 // NewLevelDB instantiate a new LevelDB instance according to DBBackend.
 func NewLevelDB(name, dir string) (db dbm.DB, err error) {
 	backend := dbm.GoLevelDBBackend
-	if DBBackend == string(dbm.CLevelDBBackend) {
```

Comment:
This should live in an init() function

Reply:
I think I've got it as you suggested. Can you check please?